src/cloud/env/variables-build.md

```diff
@@ -108,26 +108,38 @@ stage:
 
 You can configure multiple locales per theme. This customization helps speed up the build process by reducing the number of unnecessary theme files. For example, you can build the _magento/backend_ theme in English and a custom theme in other languages.
 
-The following example builds the `magento/backend` theme with three locales:
+The following example builds the `Magento/backend` theme with three locales:
 
 ```yaml
 stage:
   build:
     SCD_MATRIX:
-      "magento/backend":
+      "Magento/backend":
         language:
           - en_US
           - fr_FR
           - af_ZA
 ```
+The following example builds the three themes with three locales:
+
+```yaml
+stage:
+  build:
+    SCD_MATRIX:
+          {
```

Comment:
This format is inconsistent with the rest of the page, other areas of Cloud documentation and also the .magento.env.md examples. this could cause confusion

Please rewrite to use the standard format (as in lines 117 - 121)

Reply:
Hello @BarnyShergold
Thank you for the suggestion, I have updated PR, can you review it?